graphql.go

```diff
@@ -100,6 +101,23 @@ func MaxDepth(n int) SchemaOpt {
 	}
 }
 
+// MaxQueryComplexity specifies the complexity a query.
```

Comment:
(nit) "... the complexity **of** a query."

Reply:
I fixed. Thanks.